0000-interledger-protocol-4/0000-interledger-protocol-4.md

```diff
@@ -14,22 +14,22 @@ Interledger is a protocol for sending packets of money across different payment
 - **End-to-End Principle** - Inspired by the Internet, any features that do not need to be implemented by the core Interledger Protocol and network of connectors should be built into the edges of the network (the sender and receiver).
 
 ### Terminology
 
-- **Account** - Two peers establish an account with one another where they track the current obligations they hold with one another; i.e. the balance on the account. The account is denominated in a single asset. The peers agree on the precision and scale they will use for the amounts of that asset when they express amounts in messages between one another. Accounts may be pre-funded (in which case the customer or peer trusts the connector for the value of the account balance) or post-funded (in which case the connector trusts the customer or peer for the amount the account is allowed to go negative). Connectors may refuse to process further ILP packets if an account balance goes too low. Note that the mentions of "accounts" in this document refers to those that Interledger participants _hold with one another_, rather than the "accounts" they may hold on underlying ledgers because the latter are not directly part of the ILPv4 packet flow. 
+- **Account** - Two peers establish an account with one another to track the current obligations they hold with one another. The account balance represents the difference between the value of ILP packets sent and received and the amount of value that has been transferred through an underlying ledger or payment channel (for example, if peer A has signed payment channel updates for 100 units to peer B, peer A sent ILP packets worth 150 units through B, A received ILP packets worth 30 units from B, then peer A owes 20 units to peer B). Each account is denominated in a single asset and peers agree on the precision and scale they will use for the amounts of that asset when they express amounts in messages between one another (for example, a scale of 9 means that an amount of 1000000000 in an ILP packet sent between those peers represents 1 unit of that asset). Accounts may be pre-funded (in which case the customer or peer trusts the connector for the value of the account balance) or post-funded (in which case the connector trusts the customer or peer for the amount the account is allowed to go negative). Connectors may refuse to process further ILP packets if an account balance goes too low. Note that the mentions of "accounts" in this document refers to those that Interledger participants _hold with one another_, rather than the "accounts" they may hold on underlying ledgers because the latter are not directly part of the ILPv4 packet flow.
```

Comment:
I think it's still unclear that WHO has the responsibility to maintain the account, and WHERE to store it.